scripts/build_test.xml

```diff
@@ -97,13 +97,24 @@
 	</target>
 
 	<target name="stage_test_material" depends="-create_test_directory">
-		<copy todir="${ascii.test.dir}">
-			<fileset dir="${ascii.workspace.dir}">
-				<include name="TKG/**" />
-				<include name="*.mk" />
-			</fileset>
-		</copy>
-
+		<if>
+			<equals arg1="${os.name}" arg2="z/OS"/>
```

Comment:
Space before `/`, please, like lines 105, 106, 112, 113.

Reply:
Sorry missed that, added now. Thank you.